Formula/eye-d3.rb

```diff
@@ -61,19 +61,20 @@ class EyeD3 < Formula
     sha256 "b3bda1d108d5dd99f4a20d24d9c348e91c4db7ab1b749200bded2f839ccbe68f"
   end
 
-  def install
-    python3 = "python3.11"
-    venv = virtualenv_create(libexec, python3)
-    venv.pip_install resources
+  # Switch build-system to poetry-core to avoid rust dependency on Linux.
```

Comment:
so no `poetry` build dependency declaration?

Reply:
yea this avoids needing  `poetry` like we do elsewhere (https://github.com/python-poetry/poetry-core is self-contained)

previously, we were executing a poetry-generated [`setup.py`](https://github.com/nicfit/eyeD3/blob/0.9.x/setup.py) so poetry wasn't involved

Reply:
oh, i see